tasks/fork/maintain-rate.ts

```diff
@@ -85,31 +84,29 @@ HyperdriveDeployBaseTask(
             // value is not recalculated.
 
             // For EETH, we can call rebase on the Etherfi contracts using an
-            // impersonated account.
-            const config = hre.network.config as HttpNetworkUserConfig;
-            config.accounts = "remote";
-            await tc.impersonateAccount({
-                address: MAINNET_ETHERFI_MEMBERSHIP_MANAGER,
-            });
             let etherfi = await hre.viem.getContractAt(
                 "ILiquidityPool",
-                MAINNET_ETHERFI_ADDRESS,
+                EETH_LIQUIDITY_POOL_ADDRESS_MAINNET,
             );
             let etherToAdd =
                 ((await etherfi.read.getTotalPooledEther()) *
                     BigInt(interval) *
                     parseEther(rate)) /
                 (365n * 24n * BigInt(1e18));
-            await etherfi.write.rebase([etherToAdd], {
-                account: MAINNET_ETHERFI_MEMBERSHIP_MANAGER,
+            let txData = encodeFunctionData({
+                abi: (await hre.artifacts.readArtifact("ILiquidityPool")).abi,
+                functionName: "rebase",
+                args: [etherToAdd],
+            });
+            await tc.sendUnsignedTransaction({
+                from: EETH_MEMBERSHIP_MANAGER_ADDRESS_MAINNET,
+                to: EETH_LIQUIDITY_POOL_ADDRESS_MAINNET,
+                data: txData,
             });
```

Comment:
Is this what you had to do to avoid separate configs?

Reply:
Yup! Appears to have worked